src/test/ui/rfc-2008-non-exhaustive/struct.stderr

```diff
@@ -62,18 +62,33 @@ error[E0638]: `..` required with struct marked as non-exhaustive
    |
 LL |     let NormalStruct { first_field, second_field } = ns;
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+help: add `..` at the end of the field list
```

Comment:
Add some explanation along the lines of "add `..` at the end of the field list to ignore all other fields".

r=me